src/test/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpMultilineCheckTest.java

```diff
@@ -255,4 +255,15 @@ public void testGoodLimit() throws Exception {
         verify(checkConfig, getPath("InputRegexpMultilineSemantic.java"), expected);
     }
 
+    @Test
+    public void testMultilineSupport() throws Exception {
+        final DefaultConfiguration checkConfig = createModuleConfig(RegexpMultilineCheck.class);
+        checkConfig.addAttribute("format", "abc.*def");
+        checkConfig.addAttribute("matchAcrossLines", "true");
+        final String[] expected = {
+            "5: " + getCheckMessage(MSG_REGEXP_EXCEEDED, "abc.*def"),
+        };
```

Comment:
Is there something wrong that it isn't matching multiple times?

Reply:
@pbludov Suppose I wanted it to match each case in the file. How would I make it work?

Reply:
Your pattern is greedy. It matches one fragment:
```
// abc
// def
// abc
    }

    void method2() {
// def
```
After switching the test to lazy quantifier `abc.*?def` it reports two violations:
```
InputRegexpMultilineMultilineSupport.java:5: Line matches the illegal pattern 'abc.*?def'. 
InputRegexpMultilineMultilineSupport.java:7: Line matches the illegal pattern 'abc.*?def'. 
```

So it is a correct behavior. But your are right, it is not obvious. Please add an example to the docs for this.

Reply:
Done and thanks.